cassandra/src/main/scala/com/stratio/crossdata/connector/cassandra/DefaultSource.scala

```diff
@@ -232,7 +242,7 @@ object DefaultSource {
   val CassandraConnectionHostProperty = "spark_cassandra_connection_host"
   val CassandraDataSourceProviderPackageName = DefaultSource.getClass.getPackage.getName
   val CassandraDataSourceProviderClassName = CassandraDataSourceProviderPackageName + ".DefaultSource"
-
+  val CassandraDataSourcePrimaryKeyNameProperty ="primary_key"
```

Comment:
Instead of using this property, the creation options should require, at least, the partition key, and the clustering key as an optional parameter. Thus, the type of the Primary key can be inferred.